### ISSUE TITLE
One-click option to enable services blocking on one device - Adguard home in docker

### ISSUE THREAD

**Issue by uroshj** [AdguardTeam/AdGuardHome#7509]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### The problem

My son watches youtube for xx amount of minutes on the TV.
I am able to login to Adguard, select the device and disable standard blocking for the device (youtube already selected). That way, youtube stops working on the device.
Now, to explain what to do to my wife - no, I would rather have a simpler option.

### Proposed solution

If there would be an webservice call like: ip/settings/device=xx/clokingenable=true (or whatever that would look lie)
Another (better) option would be, if I could integrate that in HomeAssistant. Either over the add-on, or as a webservice that I could call simply with a click of a button. That is something my wife can then also do.
![image](https://github.com/user-attachments/assets/7ef79497-a28a-407a-b44a-1e317fb79fcd)
Not sure, If that can also be added to filtering
![image](https://github.com/user-attachments/assets/ec1413bc-5743-4637-a29d-3e1348075d71)

### Alternatives considered and additional information

In worst case - if I can integrate the block into the parental control - I can trigger that already in HomeAssistant, sadly I have not found on how to add something into the Parental control. Sure, I do understand, that in this case youtube would be blocked for all devices, but for a quick fix, that would also work. Later it would be really cool, if such an option on a device level would be there.

**Comment by Dynamic5912** [comment]
Can you not use the scheduled time instead so it's a regular allow/block?

**Comment by uroshj** [comment]
> Can you not use the scheduled time instead so it's a regular allow/block?

No, I would need it to be enabled/disabled by button. As needed. So no schedules are possible.

**Comment by Dynamic5912** [comment]
Then the only way to do this currently is by logging into the AGH dashboard

**Comment by uroshj** [comment]
Thanks for the reply.
Would there be an option, to add the youtube block to parental control? That I could select which services should be blocked wit parental control? I know, that would be for all, but still better.
Would there be an option to add something like this to the to-do list?
Thanks!

**Comment by Dynamic5912** [comment]
Parental control web service?

**Comment by uroshj** [comment]
I can enable it simply in HA:
![image](https://github.com/user-attachments/assets/eb534d29-e9cd-4a15-9321-e320eff5bcb3)
So if I could add YouTube to it (I did not see any option to change what is assigned there), then my wife would be able to turn that on (YouTube would be blocked on the whole network).
Sure it is not exactly what I would like to achieve (whole network instead of one/more devices), but it is still better than explaining where she has to go in, what to select, etc over the web access in AdGuard home.

**Comment by Dynamic5912** [comment]
You can change that service and what it blocks.

**Comment by uroshj** [comment]
> You can't change that service and what it blocks.

That is exactly my point :)

Maybe it would be good also to consider such use cases, as I hope, I am not the only one with such ideas :)
Maybe also in the add-on in HA. For the "non-technical" users :)

Thanks either way!

**Comment by uroshj** [comment]
Yes, I know. Would be a workaround, just to be able to block it quick.

**Comment by Dynamic5912** [comment]
> Yes, I know. Would be a workaround, just to be able to block it quick.

It's not a workaround as it blocks other things as well.

**Comment by uroshj** [comment]
Not a problem if adult content is blocked for that time.

**Comment by Dynamic5912** [comment]
YouTube doesn't fall under the remit of parental control though.

Have you looked at the unofficial mobile apps?

They offer options to enable and disable services per client when purchased.

**Comment by ainar-g** [comment]
Sorry, I'm not entirely sure what this feature request is about.  There is a feature request for custom UI elements in #1502.  In the meantime, if you don't want to use the current UI, and you can code, you could call the [documented API](https://github.com/AdguardTeam/AdGuardHome/blob/21945c6058ed0227b741d1ac1fd0230b7e1e3dc6/openapi/openapi.yaml) directly.

I'm going to close this one as a duplicate of #1502, and please let me know if I didn't get your request correctly.
